api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -6,18 +6,12 @@
                     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified">
     <addColumn tableName="user">
-      <column name="creation_time" type="datetime">
```

Comment:
Has this already been run? If so, do we know it's safe to change it? We don't want to lose the ability to keep all the environments in lockstep.

In general, if I'm looking at the history of a liquibase xml file, I only ever expect to see changes when a new step is added, and the individual step xmls should only have one commit, right?

Reply:
See the slack thread on why it's gotten so complicated but the TLDR on our approach is that we are going to manually rollback the 115 changset that ran on test so that this version can successfully execute.